AppWidget/app/src/main/res/xml/app_widget_weather_forecast_glance.xml

```diff
@@ -16,10 +16,10 @@
 <appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
     android:description="@string/app_widget_weather_description"
     android:initialLayout="@layout/widget_loading"
-    android:minWidth="110dp"
-    android:minHeight="96dp"
-    android:minResizeWidth="96dp"
-    android:minResizeHeight="64dp"
+    android:minWidth="184dp"
```

Comment:
I think this looks good, but I recommend to try if the default size looks good with a API level that doesn't use targetCell(Width/Height).

Reply:
Yup. I checked that and it's fine